src/sqlfluff/api/simple.py

```diff
@@ -171,3 +171,22 @@ def parse(
     if parsed.tree is None:  # pragma: no cover
         return {}
     return parsed.tree.as_record(show_raw=True)
+
+
+def reflow(
+    sql: str,
+    dialect: str = "ansi",
+    config_path: Optional[str] = None,
+) -> Dict[str, Any]:
+    """Parse and reflow the tree in a consistent fashion."""
```

Comment:
I feel like this description should be explicit that it's reflowing _whitespace_ while leaving code untouched.